module/VuFindSearch/tests/unit-tests/src/VuFindTest/Backend/Solr/BackendTest.php

```diff
@@ -644,10 +684,19 @@ protected function loadResponse($fixture)
     protected function getConnectorMock(array $mock = [], $client = null)
     {
         $map = new HandlerMap(['select' => ['fallback' => true]]);
-        $client = $client ?? new \Laminas\Http\Client();
         return $this->getMockBuilder(\VuFindSearch\Backend\Solr\Connector::class)
             ->onlyMethods($mock)
-            ->setConstructorArgs(['http://example.org/', $map, $client])
+            ->setConstructorArgs(
+                [
+                    'http://localhost/',
+                    $map,
+                    function () use (&$client) {
```

Comment:
Is there a reason you're passing a reference here, but not in WriterTest? Just curious about the apparent inconsistency.

Reply:
Just a mistake.